docker/docker-compose.yml

```diff
@@ -1,34 +1,39 @@
-version: '3.2'
+version: "2.4"
 ### OMS STATUTORY     #######################################
-### mongodb Container #######################################
 services:
     postgres-oms-statutory:
         restart: always
         image: postgres:10
         volumes:
             - postgres-oms-statutory:/var/lib/postgresql/data
         expose:
-            - 5432
+            - "5432"
         environment:
-            POSTGRES_USER: postgres
-            POSTGRES_PASSWORD: ${PW_POSTGRES}
+            POSTGRES_USER: "postgres"
+            POSTGRES_PASSWORD: "${PW_POSTGRES}"
+
     oms-statutory:
         restart: always
-        build:
-            context: ./$PATH_OMS_STATUTORY/..
-            dockerfile: ./docker/oms-statutory/Dockerfile.dev
-        image: aegee/oms-statutory:dev
+        image: aegee/oms-statutory:stable
         volumes:
             - oms-statutory-media:/usr/app/media
-            - ./$PATH_OMS_STATUTORY/../:/usr/app/src
             - shared:/usr/app/shared:ro
-        links:
+        depends_on:
             - postgres-oms-statutory
         expose:
-            - 8084
+            - "8084"
         environment:
-            BUGSNAG_KEY: a7dfed96a71033eca7f4469e8f88878b
-            PG_PASSWORD: ${PW_POSTGRES}
+            BUGSNAG_KEY: "${BUGSNAG_KEY_STATUTORY}"
+            PG_PASSWORD: "${PW_POSTGRES}"
+            NODE_ENV: "${MYAEGEE_ENV}"
```

Comment:
Also this: afaik the convention on naming the environments for node and elixir are different, it's dev/prod for elixir and development/production for node. Not sure if there's a way to do it under one name

Reply:
I don't think it's a difference of convention, i think one can either use short version or long version and as usual we didn't talk about it at the beginning - so the quickest thing to do is to change the code of elixir and accept the name long, OR go through your code and make it shorter. For my testing in staging, I'm gonna disable the mailer and change the core because I don't want to waste another 5 hours going through all repos and making PRs

Reply:
I can update it in my apps if needed, it's not a big thing to do, only thing required is to change config.js